test/kuttl/tests/deploy/deployment/configure_manila_share_backend.yaml

```diff
@@ -21,3 +21,20 @@ spec:
          cephfs_auth_id=openstack
          cephfs_cluster_name=ceph
          cephfs_protocol_helper_type=CEPHFS
+  extraMounts:
```

Comment:
Should we also change the associated assert [1] to match that `extraMounts` are actually present in the resulting CR?
Not sure you want to do in the same PR, but as a follow up we can try an extra kuttl step in the first assert that uses the `manila` client to check `service list` and match the fact that `cephfs` is there.

[1] https://github.com/openstack-k8s-operators/manila-operator/blob/main/test/kuttl/common/assert_sample_deployment.yaml

Reply:
ack on both these suggestions; thank you; will update this PR and include those

Reply:
I think we should do the check I mentioned in my previous comment in a follow up, so we can unblock CI in the short term.